### DISCUSSION TITLE
Contents Misaligned When Icons Shown

### DISCUSSION THREAD

**Discussion by theodorebaltis** [eza-community/eza#1012] [Development]
I'm having an issue with alignment similar issue to https://github.com/eza-community/eza/issues/556

I am running a Windows MSYS2 shell.
When running in the included Mintty terminal I experience the troublesome behavior immediately.
However, when running in Windows Terminal I only experience this behavior after I open a tmux session first.

I didn't want to open an issue before getting some feedback from the community/developers.

Notes:
Same behavior with 0.18.16 and 0.17.1
Same behavior with several nerd fonts

**Comment by gierens** [comment]
Hi @theodorebaltis 

Hm, I'm not experienced with Windows but I also think we would need to see some screenshots anyway to make a guess where this is coming from.

In any case this definitely sounds like something better suited for an issue, feel free to open one.
